session/types.proto

```diff
@@ -117,8 +117,15 @@ message SessionToken {
   neo.fs.v2.refs.Signature signature = 2 [json_name = "signature"];
 }
 
-// Extended headers for Request/Response. May contain any user-defined headers
-// to be interpreted on application level.
+// Extended headers for Request/Response may contain any user-defined headers to
+// be interpreted on application level. It may be used by Storage Nodes for
+// experimental protocol extensions with official adoption of such changes in
+// the future protocol versions.
+//
+// It may be useful to have extended headers in both Request and Response. For
```

Comment:
This sentence is a bit confusing for me. It "**might be** useful" but, "Response **should have**". So is it strictly required by the spec or just a recommendation?

Reply:
@EliChin we need help in fixing the language here. I wanted to say that it's logical to make response message have an indication of some extended header used during request processing. There may be cases when this makes no sense, like asking to log the request or similar "invisible" actions, so it's not a strict requirement.